demo/px-vis-paired-bar-demo.html

```diff
@@ -51,7 +51,7 @@
       <px-demo-props slot="px-demo-props" props="{{props}}" config="[[chosenConfig]]"></px-demo-props>
 
       <iron-ajax
-        url="[[importPath]]]data.json"
+        url="/demo/data.json"
```

Comment:
I think this'll break on predix-ui.com, since this is an absolute path and `https://predix-ui.com/demo/data.json` doesn't exist.